rsconf/component/__init__.py

```diff
@@ -614,7 +614,7 @@ def _find_tls_crt(j2_ctx, domain):
     from rsconf import db
 
     d = j2_ctx.rsconf_db.tls_d
-    for crt, domains in j2_ctx.component.tls_crt.items():
+    for crt, domains in j2_ctx.component.get("tls_crt", PKDict()).items():
```

Comment:
Why is this change necessary?

Reply:
I agree that tls_crt should not be required and tls_crt_create is now defaulted as well. 

ac00b67 is a "proper" fix. I don't like implicitly defining defaults in some low level module. I like them initialized in in compile or in this case. I think defaults like this should be where all the other defaults are initialized. component doesn't get access to Host and it would break the abstraction to have db import component so I POSITed the defaults in db.

LMK

Reply:
Ya much better fix. Thanks.